docs/quickstart.md

```diff
@@ -0,0 +1,760 @@
+Also, we added boilerplate code and even with additional third party library it is not straightforward. 
+Can we do better? Yes! We can use powertools logger capabilities as with next example.
+=== "app.py"
+
+    ```python hl_lines="1 6 13 19 23"
+    from aws_lambda_powertools import Logger
+    from aws_lambda_powertools.logging import correlation_paths
+
+    from aws_lambda_powertools.event_handler.api_gateway import ApiGatewayResolver
+
+    logger = Logger(service="APP")
+
+    app = ApiGatewayResolver()
+
+
+    @app.get("/hello/<name>")
+    def hello_name(name):
+        logger.info(f"Request from {name} received")
+        return {"statusCode": 200, "body": {"message": f"hello {name}!"}}
+
+
+    @app.get("/hello")
+    def hello():
+        logger.info(f"Request from unknown received")
+        return {"statusCode": 200, "body": {"message": "hello unknown!"}}
+
+
+    @logger.inject_lambda_context(correlation_id_path=correlation_paths.API_GATEWAY_REST, log_event=True)
+    def lambda_handler(event, context):
+        return app.resolve(event, context)
+    ```
+
+We imported powertools Logger class which we created instance from in line 6. And this is basically it! By using powertools logger we have all required configuration done out of the box. 
+We used `logger.inject_lambda_context` decorator to inject lambda context into every log. We also instruct logger to log correlation id taken from API Gateway and to log coming event automatically. 
+Correlation id will help with tracking request between api gateway and lambda.
+
+As a result, we should see logs with following attributes.
+=== "Example Application Structured Log"
+```json
+{
+    "level":"INFO",
+    "location":"hello:17",
+    "message":"Request from unknown received",
+    "timestamp":"2021-10-22 16:29:58,367+0000",
+    "service":"hello",
+    "sampling_rate":"0.1",
+    "cold_start":true,
+    "function_name":"HelloWorldFunction",
+    "function_memory_size":"256",
+    "function_arn":"arn:aws:lambda:us-east-1:123456789012:function:HelloWorldFunction",
+    "function_request_id":"d50bb07a-7712-4b2d-9f5d-c837302221a2",
+    "correlation_id":"bf9b584c-e5d9-4ad5-af3d-db953f2b10dc"
+    }
+```
+By having structured logs like this, we might easily search and analyse them in [CloudWatch Logs Insight](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/AnalyzingLogData.html).
+=== "CloudWatch Logs Insight Example"
+![CloudWatch Logs Insight Example](./media/cloudwatch_logs_insight_example.png)
+## Tracing
+Next task you decided to work on is to add proper tracing mechanism to your stack. Developers want to be able to analyse request traces coming through API Gateway to your Lambda. 
+Together with structured logs it will be a great step to providing observability for your application!
+Ideal service for this purpose is [AWS X-RAY](https://aws.amazon.com/xray/). How can we send traces from our application to the AWS X-RAY service?
```

Comment:
Reduce marketing, ideal is a strong word 😃

Reply:
True :-) Rephrased to:
`The AWS service that has these capabilities is [AWS X-RAY](https://aws.amazon.com/xray/). How do we send application trace to the AWS X-RAY service then?
Let's first explore how we can achieve this with [x-ray SDK](https://docs.aws.amazon.com/xray-sdk-for-python/latest/reference/index.html), and then try to simplify it with the Powertools library.`